dts/bindings/dma/st,stm32-dma-v2.yaml

```diff
@@ -17,35 +17,62 @@ description: |
     this value is 0 for Memory-to-memory transfers
     or a value between <1> .. <dma-generators> (not supported yet)
     or a value beweeen <dma-generators>+1  ..  <dma-generators>+<dma-requests>
-    3. channel-config: A 32bit mask specifying the DMA channel configuration
-    which is device dependent:
-        -bit 6-7 : Direction  (see dma.h)
-               0x0: MEM to MEM
-               0x1: MEM to PERIPH
-               0x2: PERIPH to MEM
-               0x3: reserved for PERIPH to PERIPH
-        -bit 9 : Peripheral Increment Address
-               0x0: no address increment between transfers
-               0x1: increment address between transfers
-        -bit 10 : Memory Increment Address
-               0x0: no address increment between transfers
-               0x1: increment address between transfers
-        -bit 11-12 : Peripheral data size
-               0x0: Byte (8 bits)
-               0x1: Half-word (16 bits)
-               0x2: Word (32 bits)
-               0x3: reserved
-        -bit 13-14 : Memory data size
-               0x0: Byte (8 bits)
-               0x1: Half-word (16 bits)
-               0x2: Word (32 bits)
-               0x3: reserved
-        -bit 15: Reserved
-        -bit 16-17 : Priority level
-               0x0: low
-               0x1: medium
-               0x2: high
-               0x3: very high
+    3. channel-config: A name custom DMA flags for channel configuration
+    which is device dependent see stm32_dma.h:
+       /** DMA  cyclic mode config on bit 5*/
+       #define STM32_DMA_CH_CFG_MODE(val)		((val & 0x1) << 5)
+       #define STM32_DMA_MODE_NORMAL		STM32_DMA_CH_CFG_MODE(0)
+       #define STM32_DMA_MODE_CYCLIC		STM32_DMA_CH_CFG_MODE(1)
```

Comment:
I don't see great interest in copy/pasting headers. People can open them directly.

Reply:
ok